doc/source/cross-language.rst

```diff
@@ -0,0 +1,246 @@
+Cross Language
+==============
+
+This page will cover some more cross language examples of using Ray's flexible programming model.
+
+Python calling Java
+-------------------
+
+You can write a Java class as follows:
+
+.. code-block:: java
+
+  // A regular Java class.
+  public class Counter {
+
+    private int value = 0;
+
+    public int increment() {
+      this.value += 1;
+      return this.value;
+    }
+
+    public static int add(int a, int b) {
+      return a + b;
+    }
+  }
+
+Then you can create the Java actor from Python, or call Java remote function
+from Python:
+
+.. code-block:: python
+
+  import ray
+
+  ray.init(_include_java=True, _load_code_from_local=True)
+
+  # Define a Java class.
+  counter_class = ray.java_actor_class(
+        "<your java package>.Counter")
```

Comment:
we can add something like `package io.ray.demo` in the Java example, and use the real package here.

Reply:
Good suggestion!